platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/config/internal/PlatformConfigUtils.java

```diff
@@ -66,4 +67,38 @@ public static void logNotKnownConfigProperties(@NonNull final Configuration conf
                     logger.warn(STARTUP.getMarker(), message);
                 });
     }
+
+    /**
+     * Logs all applied mapped properties. And suggests to change the new property name.
+     */
+    private static void logAppliedMappedProperties(@NonNull final Set<String> configNames) {
+        final Map<String, String> mappings = ConfigMappings.MAPPINGS.stream()
+                .collect(Collectors.toMap(ConfigMapping::originalName, ConfigMapping::mappedName));
+
+        configNames.stream().filter(mappings::containsKey).forEach(name -> {
+            final String message = ("Configuration property '%s' was mapped to '%s'. "
+                            + "Consider change the new property name")
+                    .formatted(name, mappings.get(name));
+
+            logger.info(STARTUP.getMarker(), message);
```

Comment:
```suggestion
            logger.warn(STARTUP.getMarker(), message);
```

Reply:
Also, I'd consider upgrading the message for configurations that don't match anything to an error (using the `EXCEPTION` marker). IMO having a mistyped setting should be taken very seriously.

Reply:
Done